.github/workflows/dev_deploy.yml

```diff
@@ -1,19 +1,26 @@
 name: Deploy to Dev
-on: 
+on:
   workflow_dispatch:
     inputs:
       environment:
         description: "Deploy to dev"
         default: dev
+        type: environment
         required: true
       ref:
         description: "Branch or Commit"
         default: master
         required: true
-  
+      lambdaDeploy:
+        description: "Include Lambda in deployment?"
+        default: false
+        required: false
+        type: boolean
+
 jobs:
   setup-environment:
     runs-on: ubuntu-latest
+    environment: ${{ github.event.inputs.environment }}
```

Comment:
Why does this use `github.event.inputs.environment` but others do not? Is the full path for "inputs" never required? Everywhere in lambda-functions.yml it cuts out `github.event.inputs` and just uses `inputs.`.

Reply:
This is all how it pulls from the `context` object inside the actions pipeline. 

If the workflow is a `workflow_call` then you can just pull `inputs` directly and it references the context. If you use the `workflow_dispatch` then you use to have to reference the `github.event` object to pull the inputs from the event that triggered it. 

I think this has been updated to where you "can" use the `context` inputs no matter where but I still like to use the event inputs on the `workflow_dispatch` to make sure I'm really getting what I want. Since the `workflow_call` doesn't trigger it won't have that specific event but is called by another one.